downloads/swan-lake-release-notes/swan-lake-alpha3/RELEASE_NOTE.md

```diff
@@ -2,41 +2,931 @@
 layout: ballerina-blank-page
 title: Release Note
 ---
-### Overview of jBallerina {{ site.data.stable-latest.metadata.version }}
-The jBallerina {{ site.data.stable-latest.metadata.version }} patch release improves upon the {{ site.data.stable-latest.metadata.version }} release by introducing the features listed below and addressing a number of [bugs](https://github.com/ballerina-platform/ballerina-lang/issues?q=is%3Aissue+milestone%3A%22Ballerina+{{ site.data.stable-latest.metadata.version }}%22+label%3AType%2FBug+is%3Aclosed) and [improvements](https://github.com/ballerina-platform/ballerina-lang/issues?q=is%3Aissue+milestone%3A%22Ballerina+{{ site.data.stable-latest.metadata.version }}%22+is%3Aclosed+label%3AType%2FImprovement).
+### Overview of Ballerina Swan Lake Alpha3
+
+This Alpha3 release includes the language features planned for the Ballerina Swan Lake release. Moreover, this release includes improvements and bug fixes to the compiler, runtime, standard library, and developer tooling. This release note lists only the features and updates added after the Alpha2 release of Ballerina Swan Lake.
```

Comment:
I believe we decided to use "The <VERSION> release". https://github.com/ballerina-platform/ballerina-release/pull/109/files#diff-5ff698e00894e1160e7ae1009a4cffaa894c9fb736b9000d9c4ac07da5be04faR7

Shall we also include "Ballerina Swan Lake"

```suggestion
The Ballerina Swan Lake Alpha3 release includes the language features planned for the Ballerina Swan Lake release. Moreover, this release includes improvements and bug fixes to the compiler, runtime, standard library, and developer tooling. This release note lists only the features and updates added after the Alpha2 release of Ballerina Swan Lake.
```